project/Dependencies.scala

```diff
@@ -175,7 +175,7 @@ object Dependencies {
   val workbenchServiceTest: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-service-test" % serviceTestV % "test" classifier "tests" excludeAll (excludeGuava)
 
```

Comment:
Any dependencies we can remove, like `spray-json`? :)

Reply:
I'd love to...but `TestProxy` still uses spray (there could be others too), so I didn't spend more time on this...

Reply:
removed spray json from `http` module...it still exists for `automation`, didn't feel like introducing more changes in this PR tho since it's already quite big

Reply:
turns out we can't remove sprayjson just yet because `HttpGoogleIamDAO` replies on it